gorocksdb.h

```diff
@@ -28,3 +28,17 @@ extern void gorocksdb_mergeoperator_delete_value(void* state, const char* v, siz
 /* Slice Transform */
 
 extern rocksdb_slicetransform_t* gorocksdb_slicetransform_create(uintptr_t idx);
+
+/* Statistics/Tickers */
+#ifdef __cplusplus__
+extern "C" {
+#endif
```

Comment:
Should it follow same pattern as above for exporting functions?

Reply:
The extern "C" wasn't required until I added in C++ code.  I moved the extern "C" to the top of the file to make things more consistent.  Without this, cgo will fail to link in the functions in the C++ file.